api/src/Controller/Report/ReportController.php

```diff
@@ -926,6 +927,9 @@ public function getReportsWithQueuedChecklists(Request $request): array
 
         $reports = [];
         foreach ($queuedReportIds as $reportId) {
+            $filter = $this->em->getFilters()->getFilter('softdeleteable');
+            $filter->disableForEntity(Client::class);
```

Comment:
Annoying to have to disable the soft delete filter to do stuff like this.
Think it's because in Digideps we still make use of things that are soft deleted, which may not be the expected behaviour when you soft delete something.

I've seen places in our code where we enable the filter again afterwards.  
[See here for example](https://github.com/ministryofjustice/opg-digideps/blob/main/api/src/Repository/ClientRepository.php#L52)

 I'm not 100% if this is needed or not. Unfortunately haven't been able to find any resources to dictate whether we need to enable afterwards or not.